netbout-web/src/main/java/com/netbout/email/BoutInviteMail.java

```diff
@@ -89,7 +89,7 @@
 
     /**
      * Send an email with register link which is like.
-     * http://www.netbout.com/b/<bout_number>?invite=<invite-key>
+     * http://www.netbout.com/b/<bout_number/>?invite=<invite-key/>
```

Comment:
@mkordas  ```@{code}``` worked I have pushed my changes for all files mentioned , if I have forgotten anything please tell.